src/main/java/org/truffleruby/interop/InteropNodes.java

```diff
@@ -1746,8 +1746,8 @@ protected boolean isJavaClassOrInterface(Object object) {
 
     }
 
-    @CoreMethod(names = "to_java_string", onSingleton = true, required = 1)
-    public abstract static class InteropToJavaStringNode extends CoreMethodArrayArgumentsNode {
+    @Primitive(name = "ruby_string_to_java_string")
```

Comment:
Seems a bit confusing name given it'll accept any interop string. Maybe just `to_java_string`?

Reply:
I named it that way to match `foreign_string_to_ruby_string`, but I can shorten it.

Reply:
Could be `interop_to_java_string` since there is already `interop_to_java_array`, etc. But it may be a little confusing given our usages here. `to_java_string` is good and simple, and reflects the node name.